spec/conman_spec.rb

```diff
@@ -61,4 +86,10 @@ def expect_field(key, value)
     expect(conman.contact_of_id(0)[key]).to eq(value)
   end
 
+  def dummies
+    contact1 = {name: "name1", address: "address1"}
+    contact2 = {name: "name2", address: "address2"}
+    contact3 = {name: "name3", address: "address3"}
+    contacts = [contact1, contact2, contact3]
```

Comment:
`contacts` is not needed here. The last line of a method in ruby is always the return value of it. Just having the array literal there is okay.